libdnf/repo/solv_repo.cpp

```diff
@@ -399,7 +475,7 @@ bool SolvRepo::load_solv_cache(const char * type, int flags) {
             }
             return true;
         } else {
-            logger.trace("Cache file \"{}\" checksum mismatch, not loading", path.native());
+            logger.trace("Not loading solvfile cache: \"{}\"", path.native());
```

Comment:
`can_use_repomd_cache()` didn't log, now we've got detailed logging in `can_use_solvfile_cache()`, and we also log the fact we are loading the cache file in the `if` branch, so I'd drop the `else` branch with the log call.

Reply:
ok, done.